command/helper/helper.go

```diff
@@ -23,6 +23,8 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+const TxTimeoutDesc = "timeout for waiting for transactions to be minted"
```

Comment:
Re-phrase it and consider renaming the const.
```suggestion
const TxTimeoutFlagDesc = "timeout for transaction processing wait"
```